ui/packages/catalog-ui-search/src/main/webapp/react-component/dropdown/dropdown.js

```diff
@@ -231,4 +259,13 @@ class Dropdown extends React.Component {
   }
 }
 
+Dropdown.propTypes = {
+  /** Turn dropdown into a controlled component and force dropdown state. */
```

Comment:
Should onOpen, onClose, and label be added to Dropdown.propTypes?

Reply:
Added